From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Mon, 3 Nov 2014 00:27:37 +0100
Subject: Remove-StyleSheet-in-ccViewer

---
 ccViewer/ui_templates/ccviewer.ui | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/ccViewer/ui_templates/ccviewer.ui b/ccViewer/ui_templates/ccviewer.ui
index 8f122d5..81a8f1b 100644
--- a/ccViewer/ui_templates/ccviewer.ui
+++ b/ccViewer/ui_templates/ccviewer.ui
@@ -20,35 +20,6 @@
    <iconset resource="../../qCC/icones.qrc">
     <normaloff>:/CC/Icons/images/icon/cc_viewer_icon_16.png</normaloff>:/CC/Icons/images/icon/cc_viewer_icon_16.png</iconset>
   </property>
-  <property name="styleSheet">
-   <string notr="true">QWidget {
-	background-color: rgb(200,200,200);
-}
-
-QMenu
-{
-	background: rgb(200,200,200);
-}
-
-QMenu::separator 
-{
-     height: 2px;
-     background: rgb(100,100,100);
-     margin-left: 10px;
-     margin-right: 5px;
- }
-
-QMenuBar 
-{
-	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(255,255,255), stop:1 rgb(200,200,200));
- }
-
-QStatusBar
-{
-	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(200,200,200), stop:1 rgb(255,255,255));
-}
-</string>
-  </property>
   <widget class="QWidget" name="centralWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
