--- a/plugins/qAnimation/QTFFmpegWrapper/ffmpegSupport.cmake
+++ b/plugins/qAnimation/QTFFmpegWrapper/ffmpegSupport.cmake
@@ -30,7 +30,7 @@ function( target_link_ffmpeg ) # 1 argum
 			elseif ( APPLE )
 				LIST(APPEND FFMPEG_LIBRARIES ${FFMPEG_LIBRARY_DIR}/lib${libfile}.dylib )
 			else()
-				LIST(APPEND FFMPEG_LIBRARIES ${FFMPEG_LIBRARY_DIR}/lib${libfile}.a )
+				LIST(APPEND FFMPEG_LIBRARIES ${FFMPEG_LIBRARY_DIR}/lib${libfile}.so )
 			endif()
 		endforeach()
 
@@ -78,4 +78,4 @@ if (WIN32)
 	endif()
 endif()
 
-endfunction()
\ No newline at end of file
+endfunction()
